packages/core/src/components/Onboarding/pages/NamePrimary.tsx

```diff
@@ -101,12 +111,22 @@ export function NamePrimary({
         </Box>
       </Box>
       <Box
+        width={{
+          base: '100%',
+          md: '60%',
+        }}
+        order={{
+          base: -1,
+          md: 0,
```

Comment:
Also wanted your thoughts on this @alexdanilowicz 
IMO, it looked better when the text was below the visual element.

Reply:
I think it looks best in all instances when the text is ABOVE and then the content, and then the buttons.

Reply:
For ex, looks weird here. Let's just be consistent and put text above in all instances, and then this is ready to merge.

<img width="285" alt="Screen Shot 2023-05-15 at 7 59 53 PM" src="https://github.com/Mirrorful/mirrorful/assets/29822597/091c2163-b740-47e7-a0c7-6d689086a29d">

Reply:
Sure. I've done that change.

The button was part of the Text Container, so I had to `absolute` position it at the bottom of the page. I hope that's fine. 
( An alternate approach could be to make the Button an sibling of the Text and Content Container and change the order depending on the device, but that would require a larger rewrite which can also affect the desktop screens )